File: 240423/곱셈의 결과/result-of-multiply.py
a=int(input())
b=int(input())
c=int(input())
lists=[0 for _ in range(10)]
num=a*b*c
while(num>0):
    lists[num%10]+=1
    num=num//10
for i in range(10):
    print(lists[i])

File: 241015/곱셈의 결과/result-of-multiply.kt
fun main()
{
    var lists=IntArray(10)
    var num=1
    for(i in 1..3)
    {
        var n=readLine()!!.toInt()
        num*=n
    }
    while(num>0)
    {
        lists[num%10]++
        num/=10
    }
    for(i in 0..9)println(lists[i])
}